pkgs/jni/test/type_test.dart

```diff
@@ -205,7 +205,8 @@ final class $FType extends JObjType<F> {
 void main() {
   if (!Platform.isAndroid) {
     checkDylibIsUpToDate();
-    Jni.spawnIfNotExists(dylibDir: "build/jni_libs", jvmOptions: ["-Xmx128m"]);
+    Jni.spawnIfNotExists(
+        dylibDir: "build/jni_libs", jvmOptions: ["-Xmx128m", "-Xcheck:jni"]);
```

Comment:
Should we make a helper method somewhere to ensure we have the same options everywhere?